### PR TITLE
👾 Finish enemy descriptions for bundle.properties [stale] ugh

### PR DESCRIPTION
**Pull request by Arkanic**
all enemies now have descriptions, and i'm pretty sure alll objects do as well.

### CODE REVIEW

File: core/assets/bundles/bundle.properties [hunk 1]
```diff
@@ -660,7 +660,6 @@ keybind.press.axis = Press an axis or key...
 keybind.screenshot.name = Map Screenshot
 keybind.move_x.name = Move x
 keybind.move_y.name = Move y
-keybind.mouse_move.name = Follow Mouse
```
Comment (matheod) [inline core/assets/bundles/bundle.properties]: Removed by mistake ?

Comment (Arkanic) [reply core/assets/bundles/bundle.properties]: oops

File: core/assets/bundles/bundle.properties [hunk 2]
```diff
@@ -1115,7 +1119,7 @@ block.sorter.description = Sorts items. If an item matches the selection, it is
 block.inverted-sorter.description = Processes items like a standard sorter, but outputs selected items to the sides instead.
 block.router.description = Accepts items, then outputs them to up to 3 other directions equally. Useful for splitting the materials from one source to multiple targets.\n\n[scarlet]Never use next to production inputs, as they will get clogged by output.[]
 block.distributor.description = An advanced router. Splits items to up to 7 other directions equally.
-block.overflow-gate.description = Only outputs to the left and right if the front path is blocked.
+block.overflow-gate.description = A combination splitter and router. Only outputs to the left and right if the front path is blocked.
```
Comment (adwitkow) [inline core/assets/bundles/bundle.properties]: This reverts #1040

File: core/assets/bundles/bundle.properties (hunk 1 above)
Comment (joshuaptfan) [reply core/assets/bundles/bundle.properties]: It wasn't removed, it was newly added today and this PR is just behind.

Comment (Arkanic) [reply core/assets/bundles/bundle.properties]: @joshuaptfan i just added

File: core/assets/bundles/bundle.properties (hunk 2 above)
Comment (Arkanic) [reply core/assets/bundles/bundle.properties]: > This reverts #1040

ill fix now

**Comment by Arkanic** [comment]
typo in new commit name :/

**Comment by itskatt** [comment]
~~Mach 1040 wow that's fast~~

File: core/assets/bundles/bundle.properties [hunk 3]
```diff
@@ -1065,6 +1065,10 @@ unit.eruptor.description = A heavy mech designed to take down structures. Fires
 unit.wraith.description = A fast, hit-and-run interceptor unit. Targets power generators.
 unit.ghoul.description = A heavy carpet bomber. Rips through enemy structures, targeting critical infrastructure.
 unit.revenant.description = A heavy, hovering missile array.
+unit.reaper.description = A superheavy air unit equipped with extremely heavy weaponry, designed brute force and decintegrate enemy defenses.
```
Comment (Paulao17) [inline core/assets/bundles/bundle.properties]:
```suggestion
unit.reaper.description = A superheavy air unit equipped with extremely heavy weaponry, designed brute force and disintegrate enemy defenses.
```

File: core/assets/bundles/bundle.properties [hunk 4]
```diff
@@ -1065,6 +1065,10 @@ unit.eruptor.description = A heavy mech designed to take down structures. Fires
...
+unit.lich.description = A heavy, flying incediary missile platform.
```
Comment (Paulao17) [inline core/assets/bundles/bundle.properties]:
```suggestion
unit.lich.description = A heavy, flying incendiary missile platform.
```

**Comment by Arkanic** [comment]
commit is stale, might remake very soon

**Comment by Anuken** [comment]
Sorry, I think I'm going to wait a bit longer to add these descriptions myself. Perhaps after I have a bit more story going.

**Comment by Arkanic** [comment]
sure

i'd like to see a nice backstory

**Pull request closed without merging**